docs/docs_settings.py

```diff
@@ -6,19 +6,6 @@
 
```

Comment:
According to the comment above, these imports will be needed when generating APIs so it would be better to keep them.

Reply:
Okay